src/cozmo/run.py

```diff
@@ -289,16 +289,17 @@ def __init__(self):
             logger.warn('No TCP connection found running Cozmo: %s' % result)
 
         android = AndroidConnector()
-        result = await self._do_connect(android, *conn_args)
-        if not isinstance(result, BaseException):
-            return result
-        logger.warn('No Android device found running Cozmo: %s' % result)
+        android_result = await self._do_connect(android, *conn_args)
+        if not isinstance(android_result, BaseException):
+            return android_result
 
         ios = IOSConnector()
-        result = await self._do_connect(ios, *conn_args)
-        if not isinstance(result, BaseException):
-            return result
-        logger.warn('No iOS device found running Cozmo: %s' % result)
+        iOS_result = await self._do_connect(ios, *conn_args)
```

Comment:
minor nitpick: we use lower snake case everywhere else for variable names so this feels slightly inconsistent